notice/1026/2016-28699.xml

```diff
@@ -18,71 +18,54 @@
-        </interpParagraph>
-        <interpParagraph label="1026-35-c-2-ii-Interp-2-ii" marker="ii.">
-          <title type="keyterm">Net decreases.</title>
-          <content>If the resulting amount calculated, after rounding, is equal to or less than the current threshold, then the threshold effective January 1 the following year will not change, but future increases will be calculated based on the amount that would have resulted.</content>
-        </interpParagraph>
-      </interpParagraph>
-    </change>
-    <change operation="modified" label="1026-35-c-2-i">
-      <paragraph label="1026-35-c-2-i" marker="(i)">
-        <content>A loan that satisfies the criteria of a qualified mortgage as defined pursuant to <ref target="USC:15-1639" reftype="external">15 U.S.C. 1639</ref>c;</content>
-      </paragraph>
-    </change>
-    <change operation="modified" label="1026-43-e-2-v-B">
-      <paragraph label="1026-43-e-2-v-B" marker="(B)">
-        <content>The <ref target="1026-2-a-11" reftype="term">consumer</ref>'s current debt obligations, alimony, and child support in accordance with appendix <ref target="1026-Q" reftype="internal">Q</ref> and paragraphs <ref target="1026-43-c-2-vi" reftype="internal">(c)(2)(vi)</ref> and <ref target="1026-43-c-3" reftype="internal">(c)(3)</ref> of this section; and</content>
-      </paragraph>
-    </change>
-    <change operation="modified" label="1026-35-c-2-ii-Interp-3">
-      <interpParagraph label="1026-35-c-2-ii-Interp-3" marker="3.">
-        <title type="keyterm">Threshold.</title>
-        <content>For purposes of § 1026.35(c)(2)(ii), the threshold amount in effect during a particular period is the amount stated below for that period.</content>
-        <interpParagraph label="1026-35-c-2-ii-Interp-3-i" marker="i.">
-          <content>From January 18, 2014, through December 31, 2014, the threshold amount is $25,000.</content>
-        </interpParagraph>
-        <interpParagraph label="1026-35-c-2-ii-Interp-3-ii" marker="ii.">
-          <content>From January 1, 2015, through December 31, 2015, the threshold amount is $25,500.</content>
-        </interpParagraph>
-        <interpParagraph label="1026-35-c-2-ii-Interp-3-iii" marker="iii.">
-          <content>From January 1, 2016, through December 31, 2016, the threshold amount is $25,500.</content>
-        </interpParagraph>
-        <interpParagraph label="1026-35-c-2-ii-Interp-3-iv" marker="iv.">
-          <content>From January 1, 2017, through December 31, 2017, the threshold amount is $25,500.</content>
-        </interpParagraph>
+    <change operation="modified" label="1026-35-c-2-ii-Interp">
+      <interpParagraph label="1026-35-c-2-ii-Interp" target="1026-35-c-2-ii">
+	<title>Paragraph 35(c)(2)(ii)</title>
+	<interpParagraph label="1026-35-c-2-ii-Interp-1" marker="1.">
```

Comment:
This element causes this notice to fail validation (via `./regml.py validate` from regulations-xml-parser), whereas it passed before. This seems to be because [the schema requires](https://github.com/cfpb/regulations-schema/blob/cfe2f07a9fcda10ff5851eeac73d6d458f85f636/src/interpretation.xsd#L109) a `<content>` element in an `interpParagraph`.

Reply:
Whoops! Fixed in latest commit.